Source/Extensions/Blazorise.DataGrid/DataGridColumn.cs

```diff
@@ -407,6 +414,16 @@ public SortDirection CurrentSortDirection
     /// </summary>
     [Parameter] public bool Filterable { get; set; } = true;
 
+    /// <summary>
+    /// Gets or sets whether the column should be used as a Group Field. A custom <see cref="GroupBy"/> function can however be provided.
+    /// </summary>
+    [Parameter] public bool Groupable { get; set; } 

```

Comment:
Also, add the Grouping parameter.

Reply:
`Groupable` vs `Grouping` seems so confusing. 
What about `Grouped` or `StartGrouped` instead of `Grouping`?

Reply:
> `Groupable` vs `Grouping` seems so confusing.
> What about `Grouped` or `StartGrouped` instead of `Grouping`?

We already have a similar naming scheme, `Editable`, `Resizable`, `Filterable`, so `Groupable` will fit. We can think of renaming `Grouping` to `Group` but I don't think it is necessary. Just document it well and it will be ok.